riposte-spi/src/main/java/com/nike/riposte/server/http/Endpoint.java

```diff
@@ -74,6 +73,19 @@ default boolean isValidateRequestContent(@SuppressWarnings("UnusedParameters") R
         return true;
     }
 
+    /**
+     * @return true if this endpoint wants validation on the {@link RequestInfo#getContent()} to ensure it always has
+     * content. If this returns true, and there was no content passed, an ApiException will be thrown to represent a
+     * bad request.
+     * <p>
+     * By default, this will check if you have a {@link #requestContentType()) defined and of a Non {@link Void} type
```

Comment:
This paragraph in general is a little clunky. Maybe:

```
By default this will return false when {@link #requestContentType()} is null or {@link Void} type (indicating you don't expect any content), true otherwise.
```

Reply:
thanks. updated!